docs/monthly-meeting/2022-08.md

```diff
@@ -0,0 +1,82 @@
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- **Discourse thread:**
+- **Calendar for future meetings:** (send your e-mail to Mariatta for a calendar invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  XXX how to edit the hackMD doc
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+- Mariatta / `@Mariatta`
+- Petr Viktorin / `@encukou`
+- Hugo van Kemenade / `@hugovk`
+- Ned Batchelder / `@nedbat`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Jim DeLaHunt / @JDLH
+- Daniele Procida
+- Pradyun Gedam
+- Erlend Aasland / `@erlend-aasland`
+
+## Discussion
+
+### How to edit hackmd doc?
+
+- Login (with GitHub or something)
+- Then use the pencil button at the top of the page
+
+### Diataxis Workshop
+
+- 70 ppl signed up, ~35 are accepted.
+- Could let a few more ppl in, but it's probably best to repeat sometime later.
+- Thanks Mariatta, Ned & CAM.
+
+### New Python Docs theme
+
+- Pradyun talks about Lutra, a new theme for "big" docs. (Pradyun's existing theme, Furo, is for "smaller docs" -- it puts everything in the sidebar)
+    -  Can we see the sample?
+    -  https://pradyunsg-cpython-lutra-testing.readthedocs.io/en/latest/
+    -  CAM: How's this different from Furo?
+        -  1. Didn't want to do too many things with Furo -- it does what it does well
+        -  2. Freedom from backwards compatibility concerns
+    -  Mariatta: Can it handle CPython docs build process?
+        -  Pradyun is looking into the docs-build script
+    -  Daniele: You showed us two different modes of Lutra
+        -  Pradyun: there are 5 modes. Configured in conf.py
+    -  Ned: the different modes seem complicated, when Python docs will use just one mode
+    -  Pradyun: I imagine python-docs-theme will customize this further, e.g. use the python.org header
+    -  Daniele: furo works for smaller sites, but its navigation model it does best and powerful, allowing the user to internalize the information architecture. In Lutra's mode, which one also prioritize docs architecture within its navigation model?
+        -  Pradyun: all of them, just exposed in slightly different ways.
+    -  CAM: 
+    -  Ned: what's the plan here? Pradyun will write a doc, etc
+        -  Pradyun: working on it, hopefully this weekend
+    -  Petr: Who will write a PEP?
+    -  Ned: Why do we need a PEP?
+    -  Daniele: when it comes to adopting diataxis, but it's a decision that should be taken after the workshops. After the workshops people will have cleaerer idea and implications of what it will mean for the docs. If there is a stron consensus, that's when we should work that decision into a PEP. It needs to be really understood by the people aking small changes. The PEP of styling is the same.
```

Comment:
```suggestion
    -  Daniele: when it comes to adopting diataxis, but it's a decision that should be taken after the workshops. After the workshops people will have clearer idea and implications of what it will mean for the docs. If there is a strong consensus, that's when we should work that decision into a PEP. It needs to be really understood by the people making small changes. The PEP of styling is the same.
```